src/vs/workbench/workbench.common.main.ts

```diff
@@ -479,4 +479,7 @@ import 'sql/workbench/contrib/extensions/browser/extensions.contribution';
 // Azure
 import 'sql/workbench/contrib/azure/browser/azure.contribution';
 
+// Onboarding
+import 'sql/workbench/contrib/onboarding/common/onboarding.contribution';
```

Comment:
This command will not have an entry point to run on normal built systems. I can doubly assure it with the environment service inside the command executor.